neuro_san_web_client/app.py

```diff
@@ -38,8 +38,9 @@
 DEFAULT_CONFIG = {
     'server_host': 'localhost',
     'server_port': 30011,
+    'web_client_host': '0.0.0.0',
     'web_client_port': 5001,
-    'agent_name': 'telco_network_support',
+    'default_agent_name': 'telco_network_support',
```

Comment:
Renamed to `default_agent_name` as it's only used when no agent name has been specified.